.hlint-universum.yaml

```diff
@@ -0,0 +1,414 @@
+
+- warn: { name: "liftIO is not needed", lhs: liftIO (swapMVar x y), rhs: Universum.swapMVar x y,
+    note: "If you import 'swapMVar' from Universum, it's already lifted" }
+
+- warn: { name: "liftIO is not needed", lhs: liftIO (takeMVar x), rhs: Universum.takeMVar x,
+    note: "If you import 'takeMVar' from Universum, it's already lifted" }
+
+- warn: { name: "liftIO is not needed", lhs: liftIO (tryPutMVar x y), rhs: Universum.tryPutMVar x y,
+    note: "If you import 'tryPutMVar' from Universum, it's already lifted" }
+
+- warn: { name: "liftIO is not needed", lhs: liftIO (tryReadMVar x), rhs: Universum.tryReadMVar x,
+    note: "If you import 'tryReadMVar' from Universum, it's already lifted" }
+
+- warn: { name: "liftIO is not needed", lhs: liftIO (tryTakeMVar x), rhs: Universum.tryTakeMVar x,
+    note: "If you import 'tryTakeMVar' from Universum, it's already lifted" }
+
+- warn: { name: "liftIO is not needed", lhs: liftIO (atomically x), rhs: Universum.atomically x,
+    note: "If you import 'atomically' from Universum, it's already lifted" }
+
+- warn: { name: "liftIO is not needed", lhs: liftIO (newTVarIO x), rhs: Universum.newTVarIO x,
+    note: "If you import 'newTVarIO' from Universum, it's already lifted" }
+
+- warn: { name: "liftIO is not needed", lhs: liftIO (readTVarIO x), rhs: Universum.readTVarIO x,
+    note: "If you import 'readTVarIO' from Universum, it's already lifted" }
+
+# IORef
+
+- warn: { name: "liftIO is not needed", lhs: liftIO (newIORef x), rhs: Universum.newIORef x,
+    note: "If you import 'newIORef' from Universum, it's already lifted" }
+
+- warn: { name: "liftIO is not needed", lhs: liftIO (readIORef x), rhs: Universum.readIORef x,
+    note: "If you import 'readIORef' from Universum, it's already lifted" }
+
+- warn: { name: "liftIO is not needed", lhs: liftIO (writeIORef x y), rhs: Universum.writeIORef x y,
+    note: "If you import 'writeIORef' from Universum, it's already lifted" }
+
+- warn: { name: "liftIO is not needed", lhs: liftIO (modifyIORef x y), rhs: Universum.modifyIORef x y,
+    note: "If you import 'modifyIORef' from Universum, it's already lifted" }
+
+- warn: { name: "liftIO is not needed", lhs: liftIO (modifyIORef' x y), rhs: Universum.modifyIORef' x y,
+    note: "If you import 'modifyIORef'' from Universum, it's already lifted" }
+
+- warn: { name: "liftIO is not needed", lhs: liftIO (atomicModifyIORef x y), rhs: Universum.atomicModifyIORef x y,
+    note: "If you import 'atomicModifyIORef' from Universum, it's already lifted" }
+
+- warn: { name: "liftIO is not needed", lhs: liftIO (atomicModifyIORef' x y), rhs: Universum.atomicModifyIORef' x y,
+    note: "If you import 'atomicModifyIORef'' from Universum, it's already lifted" }
+
+- warn: { name: "liftIO is not needed", lhs: liftIO (atomicWriteIORef x y), rhs: Universum.atomicWriteIORef x y,
+    note: "If you import 'atomicWriteIORef' from Universum, it's already lifted" }
+
+# others
```

Comment:
Did you take some old version of rules? Some of these rules are not present in latest `master`. I'd rather use latest rules, because we'll start using newest `universum` at some point where these rules don't make sense.

Reply:
I took the version you linked in https://github.com/serokell/ariadne/pull/304#discussion_r216774817.